src/pacmod3_core.cpp

```diff
@@ -807,7 +807,7 @@ void SystemCmdInt::encode(
   data[1] = cmd;
 }
 
-void SteerCmdMsg::encode(
+void SteeringCmdMsg::encode(
```

Comment:
Change `SteerRptMsg` to `SteeringRptMsg`

Reply:
I fixed it, but FYI I intend to replace all this code with auto-generated CAN parsing code in the future, so the types will match the DBC exactly.